### ISSUE TITLE
University of Wisconsin - Madison (NO-GUARANTEE label)

### ISSUE THREAD

**Issue by karusankaralingam** [CSStipendRankings/CSStipendRankings#96]
Hi all, Not sure who or what process was used to arrive at the NO-GUARANTEE label for UW-Madison. 
I am concerned and confused, especially this it has a partially verified label.

UW-Madison's graduate admissions follows a hybrid model.
a) We make some number of guaranteed aid offers (4 years for guaranteed funding through TA).
b) We make an additional small number of admissions without aid

Based on our heuristics on candidate "quality" the pool for (a) is better than the pool for (b).

What heuristic/mechanisms/document was used to determine NO-GUARANTEE? Labeling UW-Madison NO-GUARANTEE seems grossly inaccurate.

**Comment by mjc0608** [comment]
Hi, thanks for raising this issue. In short, the no-guarantee label is because of b). 

The reason is simple: asking people to do a phd while paying their own "tuition" and living expense is simply not moral. We do understand while the funding is not official guaranteed at the department level, in many case it's always provided in practice. However, we believe in such case, the department should move forward a more official guarantee.

As a result, we will keep this label until UW Madison provides some kind of funding guarantee for every incoming student, and we will add similar label for any department if some one can demonstrate there's no funding guarantee in that department. If these departments start to guarantee funding, the department should upload an official letter, with department letterhead, to our GitHub issues, and we will remove the label immediately.

We will update the website and add some clarifications and procedures regarding this label.

**Comment by mjc0608** [comment]
Besides, we are considering adding a "note" section for each department, with which people can provide more concrete information regarding different things, such as CPT-Fee and funding status. This will be complete in the near future.

**Comment by lbertge** [comment]
hello!

I am a PhD student at UW-madison. On the website, the "no-guarantee" label says it indicates 
* that this department does not guarantee any funding at all (at least for a subset of its enrolled students)

But based on the discussion above, should that instead be reworded to
* that this department does not guarantee any funding at all (for _all_ of its enrolled students)?

In any case, I also do not believe this label to be an accurate representation of the funding status for PhD candidates here. I would like to see this issue rectified, so may I clarify what it would take to remove this label?
* An official letter with department letterhead, which roughly states "this department guarantees funding, for X years, for all of its PhD admits" 
* This is to be submitted via github issue

I'm also a bit confused as to how the other schools resolved this. I tried to search all existing issues for mention of 'guarantee' and couldn't find any. I noticed some of the other schools submit an official admissions letter (with PII removed); would something like this suffice to show 'guarantee of funding for PhD students'?

thank you for your time and consideration!

**Comment by lbertge** [comment]
...nevermind, I stand corrected :/ I don't know how best to resolve this yet. 

Thanks for all the effort that goes into maintaining this website!

**Comment by mjc0608** [comment]
Hello, I understand your concern here. I believe that this website should optimize for the worst cases instead of the best case scenarios. Therefore, as long as the Set b) exists, this label will not be removed.

The case of UW-Madison is kind of special, though. I think we should probably reword the description and distinguish between PhD students and graduate students. Based on my understanding, if someone is admitted as a graduate student but needs to conduct research under a professor as a requirement, they should be considered as a PhD student. Please let me know if this is the case at your institution and if you have a better idea to reword it.

**Comment by shuyaoli** [comment]
>  Based on my understanding, if someone is admitted as a graduate student but needs to conduct research under a professor as a requirement, they should be considered as a PhD student. Please let me know if this is the case at your institution and if you have a better idea to reword it.

At UW-Madison's CS Department, all graduate students are admitted without an immediate research requirement or advisor assignment. Students with a 4-year financial aid guarantee can opt to complete a master's degree with minimal formalities (and without paying tuitions). Those without guaranteed funding can still become PhD students upon finding an advisor, with the expectation of continued funding, though not guaranteed. I suggest labeling students admitted with funding guarantee as 'PhD students' and those without as 'master students.' This approach aligns with the general distinction between PhD and master's students in other universities and accommodates the transition from master's to PhD status without insisting on a funding guarantee. 

This would imply that PhD students have funding guarantees by definition.

**Comment by mjc0608** [comment]
I will update the definition of PhD students.

One more thing to confirm, when a student admitted without funding guarantee becomes a PhD student, do they automatically get 4-year funding guarantee? If not, this students would count as "student who needs to conduct research under a professor as a requirement".

**Comment by mjc0608** [comment]
I temporally remove the no-guarantee label and put a survey there. Please forward this survey to your peer students if possible.

**Comment by shuyaoli** [comment]
Thank you for your efforts. I must express my disagreement with the 'no funding guarantee' label, as nearly all students aiming for a PhD at UW-Madison have funding guarantees, while those without are typically pursuing a master's degree. However, I genuinely appreciate your intent to consider the welfare of students in corner cases and admire the rigor in your data collection methods. 

Regarding the survey, it appears there is an issue with permissions, as it gives an error stating, 'You need permission. This form can only be viewed by users in the owner's organization.'

**Comment by mjc0608** [comment]
'You need permission. This form can only be viewed by users in the owner's organization.' -This is fixed.

I do think our labeling system is not really effective. For example, some schools and departments provide 3-year guarantee are still labeled with the guarantee, which does not make sense to me. Please let me know if you have some better ideas and you are always welcomed to contributed.

**Comment by naman** [comment]
I am a PhD student in CS at UWM. I found it really disheartening to see misguided and simply false information provided by some faculty and students in this thread while comparing the "candidate quality". 

The CS department does not make *all* offers with a "funding guarantee", and provides unfunded offers to PhD students. Therefore, the CS department does not guarantee funding for PhD students, hence the NON-GUARANTEE label was placed earlier. The department also does not guarantee the summer funding for PhD students.

Even after finding an advisor, there is NO guarantee of continued funding from the department or the advisor, as they are not legally liable to do so. This puts the burden on the PhD student to look for funding from alternate sources (other departments or advisors). Although some profs are supportive offering temp funding (or longer), this entire experience is stressful and especially worse for international students whose ability to procure funding is tied to continuing their PhD program and maintaining their F1 visa status, with no "legal" support from the department.

In my opinion, the "guarantee" on this website means a "legal guarantee" that the CS department (through an offer letter) provides to make sure that tuition and stipend finances are covered for a PhD student for a set number of years (usually, 4 in UW), which shifts the onus on the department rather than the PhD student. This means that the department is legally liable for providing a funding pathway for at least 4 years. 

tldr: CS department at UW does not guarantee funding for PhD students and does not guarantee summer funding for PhD students.

**Comment by shuyaoli** [comment]
I circulated a survey yesterday night through the UW-Madison graduate mailing list and discovered that some PhD students in our department do not have guaranteed stipends. I apologize for my previous assertion that only master's students lacked guaranteed stipends. I hope my involvement in this matter will contribute to positive changes.

**Comment by mjc0608** [comment]
@naman Thanks for the information, and our survey confirmed that. I'm adding back the no-guarantee label.

@karusankaralingam To have we remove this label, the UW Madison Department of CS would need to prove at least the following statement:
- All UW Madison CS graduate students with mandatory research requirement under an assigned or unassigned advisor for graduation has at least 4 years of funding guarantee.
